scripts/tasks/screener.ts

```diff
@@ -12,33 +11,17 @@ import { getStorybook } from '@storybook/react';
 export async function screener() {
   const screenerConfigPath = path.resolve(process.cwd(), './screener.config.js');
   const screenerConfig: ScreenerRunnerConfig = require(screenerConfigPath);
-  console.log('screener config for run');
+  console.log('screener config for run:');
   console.log(JSON.stringify(screenerConfig, null, 2));
 
-  const packageInfos = getAllPackageInfo();
-  const packagePath = path.relative(findGitRoot(), process.cwd());
-  const affectedPackageInfo = Object.values(packageInfos).find(x => x.packagePath === packagePath);
-  let affectedPackages = new Set<string>();
-  const isPrBuild = process.env.BUILD_SOURCEBRANCH && process.env.BUILD_SOURCEBRANCH.includes('refs/pull');
-
-  if (isPrBuild) {
-    affectedPackages = getAffectedPackages();
-  } else {
-    // master CI build,
-    const previousMasterCommit = getNthCommit();
-    affectedPackages = getAffectedPackages(previousMasterCommit);
-  }
-
-  debugAffectedGraph(affectedPackages);
-
   try {
-    if (!affectedPackages.has(affectedPackageInfo.packageJson.name)) {
-      await cancelScreenerRun(screenerConfig, 'skipped');
-    } else {
+    console.log(process.env.SKIPSCREENERBUILD);
+    if (process.env.SKIPSCREENERBUILD !== 'true') {
+      console.log('Running screener test:');
       const screenerStates = await getScreenerStates(screenerConfig);
       screenerConfig.states = screenerStates;
-      await screenerRunner(screenerConfig);
     }
+    await screenerRunner(screenerConfig);
```

Comment:
**question:** what is the purpose of  executing screener if  `SKIPSCREENERBUILD` is true ?

Reply:
the `screenerRunner` now calls the proxy which is the one that now decides whether a screener test should be actually run or just skipped

Reply:
```ts
    if (process.env.SKIPSCREENERBUILD !== 'true') {
      // Skipping "getScreenerStates()" as artifacts were not build
      console.log('Running screener test:');
      const screenerStates = await getScreenerStates(screenerConfig);
      screenerConfig.states = screenerStates;
    }
    
    // Run "screenerRunner" as it controls CI checks on a PR
    await screenerRunner(screenerConfig);
```

@ralucapelin Do I understand it correctly? If so, would you mind to add comments why we have these conditions?

Reply:
yes, that's correct- I will add more comments